tests/test_pytest_mock.py

```diff
@@ -162,7 +162,7 @@ def test_mock_patch_dict_resetall(mocker: MockerFixture) -> None:
     ],
 )
 def test_mocker_aliases(name: str, pytestconfig: Any) -> None:
-    from pytest_mock.plugin import _get_mock_module
+    from pytest_mock.plugin import _get_mock_module  # type: ignore
```

Comment:
the pre-commit mypy run caught these imports with `error: Cannot find implementation or library`
but now the linting test is failing because of them...any ideas?

Reply:
Not sure, but I've moved `_get_mock_module` and `parse_ini_boolean` to a `_util.py` module: this makes it clear this is not part of the API plus should fix that particular problem.